.github/workflows/create-rc-tag.yml

```diff
@@ -23,6 +23,8 @@ jobs:
     steps:
       - name: Checkout branch
         uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
```

Comment:
I can see this in the docs : 
` # The branch, tag or SHA to check out. When checking out the repository that
    # triggered a workflow, this defaults to the reference or SHA for that event.
    # Otherwise, uses the default branch.
    Ref: ''`
    
    Our default branch is `master`, When I read the runner logs it look like it's fetching the master. 
    
<img width="1402" alt="Screen Shot 2023-02-26 at 9 28 26" src="https://user-images.githubusercontent.com/100698721/221398022-fcf772cf-60f2-4fea-894c-877664957eb9.png">